### DISCUSSION TITLE
Format lines that start with numbers using gw

### DISCUSSION THREAD

**Discussion by ATM-Jahid** [LazyVim/LazyVim#2969] [Q&A] [answered]
Let's say I have the following lines in a file.
```
1 asdf
2 qwer
```
Now, I want to select these two lines and use `gw` to get something like `1 asdf 2 qwer` in a single line. I can do this with `nvim --clean` but not with LazyVim enabled. This only happens when the lines start with numbers. I think I need to disable some kind of formatting option but I don't know which one. I would appreciate any help.

**Comment by dpetka2001** [comment, marked as answer]
Remove `n` from `formatoptions`.
